packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceQueryCreatorStore.ts

```diff
@@ -109,6 +109,10 @@ export class DataSpaceQueryCreatorStore extends QueryEditorStore {
     };
   }
 
+  override get label(): string | undefined {
+    return `Create Query with DataSpace ${this.dataSpacePath}`;
```

Comment:
TODO: something to think about: in terms of styling, this is currently displayed the same way as name of a query, what should we do here?

Reply:
if you can think of a way to address this quickly, would be great to add it to this PR, else, it's okay, we can deal with it later

Reply:
yeah this was already an issue before the PR as we were messing up the styling if not an existing query. We can deal with it later